src/riak_core_metadata_evt_sup.erl

```diff
@@ -79,7 +79,13 @@ is_type_compiled(BucketType, DDL) when is_binary(BucketType) ->
     case ets:lookup(?TABLE, ?BUCKET_TYPE_PREFIX) of
         [{?BUCKET_TYPE_PREFIX, Pid}] ->
             Handlers = gen_event:which_handlers(Pid),
-            Req = {is_type_compiled, [BucketType, DDL]},
+            %% riak_ts-1.1 had in
+            %% riak_kv_metadata_store_listener:is_type_compiled a
+            %% comparison of DDL objects, so downgrade our DDL for
+            %% that (unnecessary) check to succeed:
```

Comment:
But, we still don't need to compare DDL objects, in order to just determine if it is compiled, do we?